### PR TITLE
unique objective for animal rights activists infiltrators

### PR DESCRIPTION
**Pull request by SmoSmoSmoSmok**
## About The Pull Request

 REPLACES the generic assassination objective with a CUSTOM objective U gotta INJECT the guy with a syringe a goddamn gorilla grows out of his Stomach. this gorilla is player controlled and is tasked to help the infiltratoR with his objectives it can throw its own TURD at ppl 

https://user-images.githubusercontent.com/95004236/182128096-268ee37c-c186-4232-b512-b330e85ee7a7.mp4

## Why It's Good For The Game

 it can throw its own TURD at ppl

## Changelog

:cl:
it can throw its own TURD at ppl
/:cl:

### CODE REVIEW

**Comment by Pepsilawn** [comment]
Ur the david lynch of contributors

File: fulp_modules/features/antagonists/infiltrators/albino_gorilla.dm [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+/datum/action/cooldown/mob_cooldown/charge/do_charge_indicator(atom/charger, atom/charge_target)
+	return
+
+/datum/action/cooldown/mob_cooldown/charge/Activate(atom/target_atom)
+	playsound(src, 'sound/creatures/gorilla.ogg', 200, 1)
+	return ..()
+
+/obj/projectile/gorilla_dung
+	name = "Dung pie"
```
Comment (Pepsilawn) [inline fulp_modules/features/antagonists/infiltrators/albino_gorilla.dm:30]:
```suggestion
	name = "dung pie"
```

File: fulp_modules/features/antagonists/infiltrators/albino_gorilla.dm [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+		var/mob/living/mob_target = target
+		mob_target.Paralyze(10 SECONDS)
+		mob_target.adjust_blurriness(5)
+		mob_target.visible_message(span_warning("[mob_target] is creamed by [src]!"), span_userdanger("You've been creamed by [src]!"))
+		playsound(mob_target, SFX_DESECRATION, 50, TRUE)
+		if(ishuman(mob_target))
+			target.AddComponent(/datum/component/creamed/gorilla, src)
+
+/datum/action/cooldown/spell/pointed/projectile/gorilla_dung
+	name = "Dung pie"
```
Comment (Pepsilawn) [inline fulp_modules/features/antagonists/infiltrators/albino_gorilla.dm:49]:
```suggestion
	name = "dung pie"
```

File: fulp_modules/features/antagonists/infiltrators/albino_gorilla.dm [hunk 3]
```diff
@@ -0,0 +1,76 @@
+/mob/living/simple_animal/hostile/gorilla/albino
+	name = "Albino Gorilla"
```
Comment (Pepsilawn) [inline fulp_modules/features/antagonists/infiltrators/albino_gorilla.dm:2]:
```suggestion
	name = "albino gorilla"
```

**Comment by OneAsianTortoise** [comment]
awesome and swag can't wait to throw shit at ppl

**Comment by JohnFulpWillard** [comment]
why is stuff like adding dagd not mentioned hi

**Comment by SmoSmoSmoSmok** [comment]
> why is stuff like adding dagd not mentioned hi

snneaking it past the admins please delete ur mesage

File: fulp_modules/features/antagonists/infiltrators/albino_gorilla.dm [hunk 4]
```diff
@@ -8,8 +8,8 @@
 /mob/living/simple_animal/hostile/gorilla/albino/Initialize(mapload)
 	var/datum/action/cooldown/mob_cooldown/charge/gorilla/tackle = new(src)
 	tackle.Grant(src)
-	var/datum/action/cooldown/spell/pointed/projectile/gorilla_dung/poop = new(src)
-	poop.Grant(src)
+	var/datum/action/cooldown/spell/conjure/banana/trap = new(src)
+	trap.Grant(src)
```
Comment (Pepsilawn) [inline fulp_modules/features/antagonists/infiltrators/albino_gorilla.dm:12]: soulless

**Review by Pepsilawn — CHANGES_REQUESTED** [review]

File: fulp_modules/features/antagonists/infiltrators/albino_gorilla.dm (hunk 4 above)
Comment (SmoSmoSmoSmok) [reply fulp_modules/features/antagonists/infiltrators/albino_gorilla.dm:12]: administration issue. use ur admin powers to convince the goverment  or ur just a hypocrite

**Pull request merged**